config/routes.rb

```diff
@@ -139,9 +139,9 @@
   resources :works, only: [:index, :show] do
     collection do
       get :popular
-      get ':name',
+      get ':slug',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.